libs/designsystem/card/src/card.component.ts

```diff
@@ -9,12 +9,17 @@ import {
 } from '@angular/core';
 import { ResizeObserverService } from '@kirbydesign/designsystem/shared';
 
+const KIRBY_CARD_FLAT_PROPERTY_DEPRECATION_WARNING =
+  'Deprecation warning: flat property will be removed from version 10.0. Use variant="flat" instead';
```

Comment:
As this will show up in the browsers console, it is a good idea to add some more context to the message so consumers know where to fix it when they see the message:
```suggestion
  'Deprecation warning: Setting the 'flat' property of kirby-card is deprecated and will be removed from Kirby v10. Use variant="flat" instead for identical behavior.';
```